src/utils/getDisplayName.js

```diff
@@ -0,0 +1,9 @@
+export default function getDisplayName(Component) {
+    return (
+        Component.displayName ||
+        Component.name ||
+        (typeof Component === 'string' && Component.length > 0
```

Comment:
Are there instances where we pass a string into the `getDisplayName` function? Passing in a string here and have it return the same string is basically a `toString` function.

I would prefer we could avoid the `typeof` check and just do `displayName || name || 'Unknown'`.

Reply:
Recompose has a helper for this one and we already have this dependency. https://github.com/acdlite/recompose/blob/master/docs/API.md#getdisplayname